AppointEase/AppointEase.Application/AppointEase.Application.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <ProjectReference Include="..\AppointEase.Application.Contracts\AppointEase.Application.Contracts.csproj" />
     <ProjectReference Include="..\AppointEase.Data.Contracts\AppointEase.Data.Contracts.csproj" />
+    <ProjectReference Include="..\AppointEase.Data\AppointEase.Data.csproj" />
```

Comment:
Why is the Application Layer referencing `Data `and `Data.Contracts`? If we use interfaces we don't need to reference the implementations..